pkg/apis/flowcontrol/install/install.go

```diff
@@ -36,5 +37,6 @@ func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(flowcontrol.AddToScheme(scheme))
 	utilruntime.Must(flowcontrolv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(flowcontrolv1beta1.AddToScheme(scheme))
-	utilruntime.Must(scheme.SetVersionPriority(flowcontrolv1beta1.SchemeGroupVersion, flowcontrolv1alpha1.SchemeGroupVersion))
+	utilruntime.Must(flowcontrolv1beta2.AddToScheme(scheme))
+	utilruntime.Must(scheme.SetVersionPriority(flowcontrolv1beta1.SchemeGroupVersion, flowcontrolv1beta2.SchemeGroupVersion, flowcontrolv1alpha1.SchemeGroupVersion))
```

Comment:
I expected v1beta2 to come first?

Reply:
iirc, this is what controls preferred serialization order, and we have to serialize to v1beta1 for an overlapping release

Reply:
We can remove v1alpha1, then.

Reply:
- removed `v1alpha1` from `SetVersionPriority`
- `v1beta1` is the first version in priority which keeps it the storage version.

for future, we will make `v1beta2` the storage version in 1.24, right?

Reply:
@lavalamp is there a reason we need to remove `v1alpha1`? 
let's consider the scenario where we have a cluster (upgraded from 1.22 to 1.23) and it has pre-existing `v1alpha1` objects in storage.  can the 1.23 server read the pre-existing object from storage since we removed `v1alpha1` from `SetVersionPriority`?

On the other hand, `PrioritizedVersionsForGroup` appends the `observedVersion` to the `[]schema.GroupVersion` it returns - https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/runtime/scheme.go#L592-L614. So we should be okay? please advise.

cc @MikeSpreitzer

Reply:
i have restored `v1alpha1` into `SetVersionPriority`

Reply:
OK. We have to delete it some time, but from the API first, not here.

I believe our controllers write status to both kind of objects? If so we can be very confident that alpha objects are overwritten, right?